credential/reclaim/v0.1.0/schema.json

```diff
@@ -0,0 +1,133 @@
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "proofs": {
+                    "title": "Session Id",
+                    "description": "Values of public fields defined in schema",
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "identifier": {
+                                "title": "Identifier",
+                                "description": "Identifier of provider",
+                                "type": "string"
+                            },
+                            "claimData": {
+                                "title": "Claim Data",
+                                "description": "Verifiable credential data",
+                                "type": "object",
+                                "properties": {
+                                    "provider": {
+                                        "title": "Protocol",
+                                        "description": "Protocol of request",
+                                        "type": "string"
+                                    },
+                                    "parameters": {
+                                        "title": "Parameters of request",
+                                        "description": "Stringfied JSON object of parameters and API reqeusts",
+                                        "type": "string"
+                                    },
+                                    "identifier": {
+                                        "title": "Identifier of request",
+                                        "description": "Identifier of request",
+                                        "type": "string"
+                                    },
+                                    "epoch": {
+                                        "title": "Epoch of request",
+                                        "description": "Epoch of request",
+                                        "type": "string"
+                                    }
+                                }
+                            },
+                            "signature": {
+                                "title": "Signature",
+                                "description": "The signature of verification credential",
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                }
+                            },
+                            "witnesses": {
+                                "title": "witnesses",
+                                "description": "Witness",
+                                "type": "array",
+                                "items": {
```

Comment:
You can also add a `required` under `items` to list the mandatory properties

Reply:
Done